### PR TITLE
First design of the Game and Player class (with no tests)

### PR DESCRIPTION
**Pull request by manuelkoso**

### CODE REVIEW

**Comment by manuelkoso** [comment]
There are the following TODO:

- [ ] Player method putStone
- [ ] Player method lastMove
- [ ] Player method checkFreedom
- [ ] Board method hasMoreThenOneFreeCell

File: src/main/java/Game.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        board.clearBoard();
+        gameStatus = GameStatus.STARTED;
+        while (gameStatus == GameStatus.STARTED) {
+            turn(whitePlayer);
+            turn(blackPlayer);
+        }
+        end();
+    }
+
+    void turn(Player player) {
```
Comment (manuelkoso) [inline src/main/java/Game.java:25]: I forgot the `private` keyword

File: src/main/java/Game.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
+public class Game {
+
+    private enum GameStatus {NOTSTARTED, STARTED, FINISHED}
+    private final Player whitePlayer;
```
Comment (peiva-git) [inline src/main/java/Game.java:4]: Add some annotations on the fields and methods

**Review by peiva-git — CHANGES_REQUESTED** [review]
Base classes look good, just add annotations on fields and methods where relevant

**Review by peiva-git — APPROVED** [review]
Well done

**Pull request merged**